sentry/values.yaml

```diff
@@ -1,4 +1,4 @@
-prefix:
+prefix: sentry
```

Comment:
I would like to be able to have an empty prefix. (and that should be the default value)

Sorry if I was confusing.

So you we have the default resources name :
- postgresql
- sentry-web
- snuba-web

And by applying a prefix, like sentry10 for example, it would be :
- sentry10-postgresql
- sentry10-sentry-web
- sentry10-snuba-web